common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -5188,6 +5188,11 @@
           "The SQL pipe operator syntax using |> does not support <clauses>."
         ]
       },
+      "COLLATIONS_IN_MAP_KEYS" : {
+        "message" : [
+          "Collated strings are not supported for keys of maps"
```

Comment:
Please, remove the duplicate `not supported`, see the full error message including parent's one:
```
The feature is not supported: Collated strings are not supported for keys of maps
```